### PR TITLE
create and style forgotpassword screen including forgotpasswordconfirmation…

### PR DESCRIPTION
**Pull request by juliabrowne**
… screen

### CODE REVIEW

File: js/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React, { Component } from "react";
 import AppNavigator from "./navigation/RootStackNavigator";
-import UserLoginContainer from './screens/UserLogin/UserLoginContainer'
+import ForgotPasswordContainer from './screens/ForgotPassword/ForgotPasswordContainer'
+import ForgotPassswordConfirmation from './screens/ForgotPassword/ForgotPasswordConfirmation'
```
Comment (siddharthparmar7) [inline js/App.js]: remove unnecessary imports.

File: js/components/ForgotPasswordForm/ForgotPasswordForm.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import { View, Text, TextInput, TouchableOpacity } from 'react-native'
+import getAPI from './../../config/api'
+import { Form, Field } from 'react-final-form'
+import styles from './styles'
+
+export default class ForgotPasswordForm extends Component {
+  constructor(props) {
+    super(props)
+  }
```
Comment (siddharthparmar7) [inline js/components/ForgotPasswordForm/ForgotPasswordForm.js]: remove the constructor if you are not creating a state.

File: js/components/ForgotPasswordForm/ForgotPasswordForm.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                        {...input}
+                        placeholder='Email'
+                        autoCapitalize='none'
+                        style={{ fontFamily: 'Quicksand' }}
+                      />
+                    )
+                  }}
+                />
+              </View>
+            </View>
```
Comment (siddharthparmar7) [inline js/components/ForgotPasswordForm/ForgotPasswordForm.js:55]: you have to validate user input and generate an error if there is one.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
resolve your merge conflicts so I can review this pull request. Thank you.

File: js/App.js (hunk 1 above)
Comment (juliabrowne) [reply js/App.js]: done!

File: js/components/ForgotPasswordForm/ForgotPasswordForm.js (hunk 2 above)
Comment (juliabrowne) [reply js/components/ForgotPasswordForm/ForgotPasswordForm.js]: done!

File: js/components/ForgotPasswordForm/ForgotPasswordForm.js (hunk 3 above)
Comment (juliabrowne) [reply js/components/ForgotPasswordForm/ForgotPasswordForm.js:55]: done!

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**